### PR TITLE
Update SwitchCamera.cs

### PR DESCRIPTION
**Pull request by c0ncrete**
I don't think the clip you showed me has anything to do with this script as the functionality seems quite simple. I did, however, weed out some unnecessary duplication. I'm guessing the issue has something to do with Vuforia. I'll poke around on Google and see what I can come up with.

### CODE REVIEW

**Review by mike-mirabal — CHANGES_REQUESTED** [review]
Made your changes to script in my local project. 
Built and deployed to iPhone 8+. 
Button press switches from front camera to back camera, but on the first switch, the image is still flipped as shown before on the video I sent you. Upon second button press, the flipped image becomes oriented correctly (yay! - it's possible) rather than switching back to the other camera. Subsequent button presses make no changes of the camera though. 
Basically when you press it once, it does what it did before, then when you press it again it does what it's supposed to do on the first press. Pressing yet again and repeatedly has no effect, so it remains on the selfie camera instead of switching back to default.  
Thanks! Let me know what you think.
